Dockerfile

```diff
@@ -0,0 +1,48 @@
+### Base image...
+FROM python:3-slim-buster as Base
+
+# Install poetry...
+RUN pip install poetry
+
+# Copy application code...
+COPY . .
+
+# Install prerequisites
+RUN poetry install --no-dev --no-root
+
```

Comment:
Two thoughts here:
* One is that working in the root directory doesn't cause any issues here, but is usually worth avoiding to keep your code nicely separated & easier to work with, so adding a `WORKDIR /app` for example might be worthwhile (if you do, consider how this affects my comment below on the bind mounts!)
* The other is that this currently means every app code change requires reinstalling our dependencies since the COPY needs to happen before the `poetry install ..`; we can reorganise that a little to try to take advantage of cleaner caching if we want:
```Dockerfile
# Only copy the files required to specify our dependencies
COPY pyproject.toml poetry.lock .

RUN poetry install --no-dev --no-root

# Copy everything else
COPY . . 
```